core/src/main/java/io/grpc/internal/JsonParser.java

```diff
@@ -39,6 +39,10 @@
 
   private JsonParser() {}
 
+  /**
+   * Parses a json string, returning either a {@code Map<String, Object>}, {@code List<Object>},
+   * {@code String}, {@code Double}, {@code Boolean}, or {@code Void}.
```

Comment:
s/{@code Void}/the value {@code null}/ ?

Reply:
Done.